base/applications/regedit/lang/ro-RO.rc

```diff
@@ -427,47 +427,47 @@ END
 
 STRINGTABLE
 BEGIN
-    ID_REGISTRY_MENU "Conține comenzi pentru lucrul cu întregul registru"
-    ID_EDIT_MENU "Conține comenzi pentru editarea valorilor și cheilor"
-    ID_VIEW_MENU "Conține comenzi pentru particularizarea ferestrei editorului de registru"
-    ID_FAVOURITES_MENU "Conține comenzi pentru accesarea cheilor frecvent utilizate"
-    ID_HELP_MENU "Conține comenzi pentru reprezentarea ajutorului și informațiilor despre editorul de registru"
-    ID_EDIT_NEW_MENU "Conține comenzi pentru crearea noilor chei sau valori"
+    ID_REGISTRY_MENU "Conţine comenzi pentru lucrul cu întregul registru"
+    ID_EDIT_MENU "Conţine comenzi pentru editarea valorilor şi cheilor"
+    ID_VIEW_MENU "Conţine comenzi pentru particularizarea ferestrei editorului de registru"
+    ID_FAVOURITES_MENU "Conţine comenzi pentru accesarea cheilor frecvent utilizate"
+    ID_HELP_MENU "Conţine comenzi pentru reprezentarea ajutorului şi informaţiilor despre editorul de registru"
+    ID_EDIT_NEW_MENU "Conţine comenzi pentru crearea noilor chei sau valori"
 END
 
 STRINGTABLE
 BEGIN
     ID_EDIT_MODIFY "Modifică datele unei valori"
     ID_EDIT_NEW_KEY "Adaugă o nouă cheie"
-    ID_EDIT_NEW_STRINGVALUE "Adaugă o nouă valoare șir"
+    ID_EDIT_NEW_STRINGVALUE "Adaugă o nouă valoare şir"
     ID_EDIT_NEW_BINARYVALUE "Adaugă o nouă valoare de 1 bit"
-    ID_EDIT_NEW_DWORDVALUE "Adaugă o nouă valoare de 32 de biți"
-    ID_EDIT_NEW_MULTISTRINGVALUE "Adds a new multi string value"
-    ID_EDIT_NEW_EXPANDABLESTRINGVALUE "Adds a new expandable string value"
-    ID_REGISTRY_IMPORTREGISTRYFILE "Importă registru dintr-un fișier"
-    ID_REGISTRY_EXPORTREGISTRYFILE "Exportă registrul total sau parțial într-un fișier"
-    ID_REGISTRY_LOADHIVE "Încarcă un fișier binar în registru"
-    ID_REGISTRY_UNLOADHIVE "Eliberează conținutul unui fișier binar din registru"
-    ID_REGISTRY_CONNECTNETWORKREGISTRY "Conectare la registrul unui calculator din rețea"
-    ID_REGISTRY_DISCONNECTNETWORKREGISTRY "Deconectare de la registrul calculatorului din rețea"
-    ID_REGISTRY_PRINT "Imprimă toate părțile unui registru"
+    ID_EDIT_NEW_DWORDVALUE "Adaugă o nouă valoare de 32 de biţi"
+    ID_EDIT_NEW_MULTISTRINGVALUE "Adaugă o nouă valoare cu mai multe şiruri de caractere"
```

Comment:
"de caractere" : isn't it a bit redundant, compared to the other (expanded)string translations? (Compare for example, with the "Valoare de șir &multiplu" found above.)

Reply:
Done! I rechecked the two strings and changed them as above.